src/Sentry.Maui.CommunityToolkit.Mvvm/MauiAppBuilderExtensions.cs

```diff
@@ -0,0 +1,32 @@
+using Sentry.Maui.CommunityToolkitMvvm;
+
+namespace Sentry.Maui;
+
+/// <summary>
+/// Methods to hook into MAUI and CommunityToolkit.Mvvm
+/// </summary>
+public static class MauiAppBuilderExtensions
+{
+    /// <summary>
+    /// Installs necessary services to auto-instrument CommunityToolkit.Mvvm commands
+    /// </summary>
+    /// <param name="builder">The MauiAppBuilder</param>
+    /// <returns>The MauiAppBuilder</returns>
+    public static MauiAppBuilder UseSentryCommunityToolkitIntegration(this MauiAppBuilder builder)
+    {
+        builder.Services.UseSentryCommunityToolkitIntegration();
+        return builder;
+    }
+
+
+    /// <summary>
+    /// Installs necessary services to auto-instrument CommunityToolkit.Mvvm commands
+    /// </summary>
+    /// <param name="services">The Service Collection</param>
+    /// <returns>The Service Collection</returns>
+    public static IServiceCollection UseSentryCommunityToolkitIntegration(this IServiceCollection services)
```

Comment:
question: Naming: `Use` vs `Add`.

Should we prefix this method (or even both) with `Add` instead of `Use`, since it's on the `IServiceCollection`?
To be fair, I'm a bit unfamiliar with the MAUI conventions.

Reply:
MAUI doesn't generally use the concept of builder vs post build like aspnet.  The convention is use on the maui sside